pliers/extractors/text.py

```diff
@@ -198,3 +206,95 @@ def _extract(self, stim):
         return ExtractorResult(np.array(list(data.values())).transpose(),
                                stim, self, features=list(data.keys()),
                                onsets=onsets, durations=durations)
+
+
+class WordEmbeddingExtractor(TextExtractor):
+
+    ''' An extractor that uses a word embedding file to look up embedding
+    vectors for text.
+
+    Args:
+        embedding_file (str): path to a word embedding file
+        binary (bool): flag indicating whether embedding file is saved in a
+            binary format
+    '''
+
+    _log_attributes = ('wvModel',)
+
+    def __init__(self, embedding_file, binary=False):
+        if KeyedVectors is None:
+            raise ImportError("gensim is required to create a "
+                              "WordEmbeddingExtractor, but could not be "
+                              "successfully imported. Please make sure it is "
+                              "installed.")
+        self.wvModel = KeyedVectors.load_word2vec_format(embedding_file,
+                                                         binary=binary)
+        super(WordEmbeddingExtractor, self).__init__()
+
+    def _extract(self, stim):
+        num_dims = self.wvModel.vector_size
+        if stim.text in self.wvModel:
+            embedding_vector = self.wvModel[stim.text]
+        else:
+            # UNKs will have zeroed-out vectors
+            embedding_vector = np.zeros(num_dims)
+        features = ['embedding_dim%d' % i for i in range(num_dims)]
```

Comment:
Maybe make the prefix of the embedding dimension an initializer argument, with the default set to what you hardcoded here (i.e., `"embedding_dim"`)?